### PR TITLE
Added width tokens 

### PR DESCRIPTION
**Pull request by Pallavi-PH**

### CODE REVIEW

**Comment by visualapps** [comment]
Screen Shot 2020-04-01 at 7 01 45 PM

can we fix this gap. Instead of blue can you try gray-000 or gray-100 for background

**Comment by Pallavi-PH** [comment]
> Screen Shot 2020-04-01 at 7 01 45 PM
> 
> can we fix this gap. Instead of blue can you try gray-000 or gray-100 for background

Updated

File: scss/core/tokens/_sizing.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    width: 1rem * $value !important;
+  }
+}
+
+// Maya icon sizes
+
+.dci {
+  vertical-align: middle;
+  font-size: 1em;
+}
```
Comment (visualapps) [inline scss/core/tokens/_sizing.scss:31]: icons related class could go in icons related files

File: scss/core/tokens/_sizing.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    .w-#{$size} {
+      width: 1rem * $value !important;
+      background-color: $gray-100;
+      height: 25px;
+      margin-bottom: 5px;
+    }
+  }
+  .w-auto{
+    background-color: $gray-100;
+    height: 25px;
```
Comment (visualapps) [inline scss/core/tokens/_sizing.scss:50]: w-auto comes with bootstrap units, we may not need to add it again here. Need to check.
https://getbootstrap.com/docs/4.4/utilities/sizing/#relative-to-the-parent

File: scss/core/tokens/_sizing.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  font-size: 1em;
+}
+
+@each $size, $value in $icon-size {
+  .dci-#{$size} {
+    font-size: 1em * $value !important;
+  }
+}
+
+.de-code{
```
Comment (visualapps) [inline scss/core/tokens/_sizing.scss:39]: why de-code class here

File: scss/core/tokens/_sizing.scss (hunk 2 above)
Comment (visualapps) [reply scss/core/tokens/_sizing.scss:50]: i found the same class in line number 16.

File: scss/core/tokens/_sizing.scss (hunk 3 above)
Comment (Pallavi-PH) [reply scss/core/tokens/_sizing.scss:39]: It is for displaying outside `playground`

File: scss/core/tokens/_sizing.scss (hunk 1 above)
Comment (Pallavi-PH) [reply scss/core/tokens/_sizing.scss]: Done.

File: scss/core/tokens/_sizing.scss (hunk 3 above)
Comment (Pallavi-PH) [reply scss/core/tokens/_sizing.scss]: Moved to style.scss

File: scss/core/tokens/_sizing.scss (hunk 2 above)
Comment (Pallavi-PH) [reply scss/core/tokens/_sizing.scss]: I checked. `w-auto` CSS is not defined. It might be in the new versions of bootstrap

**Pull request merged**